### DISCUSSION TITLE
[Feature] Stage Intros

### DISCUSSION THREAD

**Discussion by Amidweiz** [ikemen-engine/Ikemen-GO#828] [Feature Request]
Another feature to dump to here so it can be noted down. Stage intros which several commercial fighting games happen to have (Here is an example from KOF 95 https://youtu.be/Uk-ytRQ88Ig?t=24) in which the stage has an intro to itself before going to the round start, which these tend to be skippable. Potential implementation for this could use the preexisting BGCtrlDef in the stage def 

> [BGCtrlDef Intro]
> ctrlID = 10
>
> [BGCtrl Intro]
> type = StageIntro
> skippable = 1; Determines if the intro can be skipped or not
> mode = arcade, survival; Determines what mode that the stage intro is played on
> bgmusic = sound/whatever.mp3; Determines if there is intro music to the stage
> perRound = false; Determines if the intro is to play on every new round
> crtlID = 10 

Not sure what else could be added to this type at the moment.

**Comment by Mike77154** [comment]
Maybe download a real bout fatal Fury or a Kof stage from that classic ones that comes With the intro, I remember that I download some of the japanese autor pages, maybe this could be useful for you

**Comment by Mike77154** [comment]
But the thing of ommit this on other modes than arcade could be beatiful
